snare/cloner.py

```diff
@@ -31,12 +31,22 @@ def __init__(self, root, max_depth, css_validate):
 
     @staticmethod
     def add_scheme(url):
+        test_url = str(yarl.URL(url))
```

Comment:
What is the use of this variable?

Reply:
yarl URL cannot be accessed by index so 'test_url' has string version of the URL thus we can directly go to last index and check for presence of path.

Reply:
Yes we should check conditions only for variable 'err_url' and we do not need to check it for 'new_url'.